apps/web/lib/posthog.ts

```diff
@@ -0,0 +1,10 @@
+import { PostHog } from "posthog-node";
+
+export default function PostHogClient() {
+  const posthogClient = new PostHog(process.env.NEXT_PUBLIC_POSTHOG_KEY!, {
```

Comment:
will this fail if the env is not set?

Reply:
We aren't using this, at this time... I included it, in case we want to leverage server analytics in the future. I just added a throwable.